src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfobox.java

```diff
@@ -108,7 +113,8 @@ public WebElement getGroupHeader(int index) {
     return groupHeadersWrappers.get(index);
   }
 
-  public String getTitleTextWithIndex(int index) { return titles.get(index).getText(); }
+  public String getTitleTextWithIndex(int index) {
+    return titles.get(index).getText(); }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)